src/udev/udevadm.c

```diff
@@ -62,6 +62,9 @@ static int parse_argv(int argc, char *argv[]) {
         assert(argc >= 0);
         assert(argv);
 
+        if (argv_looks_like_help(argc, argv))
```

Comment:
This sounds like that it should be made into a verb?

Reply:
ok, using help verb as default